mantidproject/mantid
# Indirect - Automatic rqw contour plot in sqw interface

**Description of work.**
This PR rearranges the Sqw interface in Indirect Data Reduction, and there is now an automatic contour plot of rqw when a file has finished loading.

**To test:**
1.`Interfaces`->`Indirect`->`Data Reduction`->`Sqw` interface
2. There should be an empty plot with axis labels `Energy (meV)` and `Q (A-1)`.
3. Load the data below
4. After a short wait a contour plot should appear and should look like:
![image](https://user-images.githubusercontent.com/40830825/51107403-a6363c80-17e6-11e9-8c8c-3ebf485006ce.png)

**Test Files:**
[iris26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/2754736/iris26176_graphite002_red.zip)

Fixes #24458 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
